ruby/introduction/installing_ruby.md

```diff
@@ -106,7 +106,27 @@ rbenv install 3.1.2 --verbose
 
 This command will take 10-15 minutes to complete. The `--verbose` flag will show you what's going on so you can be sure it hasn't gotten stuck. While it installs, take this time to watch [this video](https://youtu.be/X2CYWg9-2N0) or to get a glass of water.
 
-When the last command is finished, set the Ruby version and verify that it's working:
+#### Troubleshooting
+
+You may get this error message:
+
+~~~bash
+ruby-build: definition not found: x.x.x
+
+See all available versions with `rbenv install --list'.
+
+If the version you need is missing, try upgrading ruby-build:
+
+  git -C /home/itorja/.rbenv/plugins/ruby-build pull
+~~~
+
+We could try running the suggested command accordingly:
```

Comment:
This sentence is a little bit clunky. Can we reword this to be a bit more direct? (Pending decisions on the first comment)